x/incentives/keeper/gauge.go

```diff
@@ -16,6 +16,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/osmosis-labs/osmosis/osmomath"
+	appparams "github.com/osmosis-labs/osmosis/v24/app/params"
```

Comment:
The changes in this file are the only actual changes. All other changes are just fixing tests to register routes so they don't fail at this step (which is good / expected behavior).